docs/docs-content/automation/palette-cli/commands/vmo.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_label: "VMO"
+title: "VMO"
+description: "Reference resource for the vmo command."
+hide_table_of_contents: false
+sidebar_position: 60
+tags: ["palette-cli"]
+---
+
+Use the `vmo` command to migrate Virtual Machines (VMs) and import and deploy vSphere Open Virtual Appliances (OVAs).
+The VMs can then be used with the Virtual Machine Orchestrator (VMO).
+
+## Subcommands
+
+- [`deploy-ova`](#deploy-ova) - Deploy an imported vSphere OVA. This command requires you to have an OVA deployment
+  file. If you do not have one, you can generate an OVA with the `import-ova` subcommand.
+
+- [`import-ova`](#import-ova) - Import a vSphere OVA. This subcommand will generate an OVA deployment configuration
+  file. The configuration can then be deployed using the `deploy-ova` subcommand.
```

Comment:
Did you have the chance to try to use the `deploy-ova` and `import-ova` commands? I ask because it would be nice to add some more context about those commands and things you may have observed.

Reply:
We are working on the guide for these, but other CLI pages simply document the commands and provide extra content in separate guides.